src/modules/Dropdown/Dropdown.js

```diff
@@ -848,12 +861,12 @@ export default class Dropdown extends Component {
   // Setters
   // ----------------------------------------
 
-  setValue = (value) => {
+  setValue = (value, resetQuery = true) => {
```

Comment:
@levithomason Thanks for a detailed feedback, I'm fully agree with your suggestions. Will make a changes soon